packages/sanity/src/desk/comments/src/types.ts

```diff
@@ -69,7 +69,11 @@ export interface CommentPath {
   field: string
 }
 
-interface CommentContext {
+/**
+ * @beta
+ * @hidden
+ */
+export interface CommentContext {
```

Comment:
Exporting this since I wanted to avoid something like `NonNullable<CommentDocument['context']>['notification']` as a return type for `getNotificationValue` – but maybe that's OK